packages/ffe-message-box/less/ffe-message-box.less

```diff
@@ -63,7 +67,7 @@
     }
 
     &__icon {
-        background-color: #FFF;
+        background-color: #fff;
```

Comment:
Så.. første gang jeg commitet rettet prettier det til `#fff`, så skulle jeg fikse en annen feil og da feila lint og ville ha `#FFF`, så da fikset jeg det for å få lint grønn og så endret selvfølgelig prettier det til `#fff` igjen etterpå 😞 

Så da antar jeg at siden prettier kjører "sist" er det den som bestemmer.

Reply:
Stryk det. Lint knekker bygget, det gjør ikke prettier så blåser i prettier her og setter `#FFF`